stdlib/http/src/main/java/org/ballerinalang/net/http/compiler/websocket/WebSocketFailoverClientResourceValidator.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http:www.apache.orglicensesLICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.net.http.compiler.websocket;
+
+import org.ballerinalang.net.http.WebSocketConstants;
+import org.ballerinalang.util.diagnostic.Diagnostic;
+import org.ballerinalang.util.diagnostic.DiagnosticLog;
+import org.wso2.ballerinalang.compiler.tree.BLangFunction;
+
+/**
+ * Service validator for failover WebSocket.
+ *
+ * @since 1.1.0
+ */
+class WebSocketFailoverClientResourceValidator extends WebSocketClientResourceValidator {
+
+    WebSocketFailoverClientResourceValidator(DiagnosticLog dlog, BLangFunction resource) {
+        super(dlog, resource);
+    }
+
+    @Override
```

Comment:
Since you are extending from `WebSocketClientResourceValidator` this method will be inherited. Hence I don't think it is necessary to override it.

Reply:
Here, I'm checking whether it has  failover client or not. So it is necessary to override it.